core/templates/dev/head/pages/practice-session-page/practice-session-page.constants.ts

```diff
@@ -21,3 +21,11 @@ oppia.constant('TOTAL_QUESTIONS', 20);
 oppia.constant(
   'PRACTICE_SESSIONS_DATA_URL',
   '/practice_session/data/<topic_name>');
+
+oppia.constant(
+  'TOPIC_VIEWER_PAGE',
+  '/topic/<topic_name>');
```

Comment:
This might fit on the above line itself I think.

Reply:
Done